S/SDPA/build_tarballs.jl

```diff
@@ -18,8 +23,11 @@ julia_version = v"1.5.3"
 # map a prerelease of 2.7.0 to 200.690.000.
 
 name = "SDPA"
-version = v"700.300.800"
 upstream_version = v"7.3.8"
+version_offset = v"0.0.0" # reset to 0.0.0 once the upstream version changes
```

Comment:
Since you're changing the compat with julia you also must change the version number, so I'd bump the offset (it's there for cases like this)
```suggestion
version_offset = v"0.0.1" # reset to 0.0.0 once the upstream version changes
```